service/src/main/kotlin/app/cash/backfila/service/scheduler/LeaseHunter.kt

```diff
@@ -54,5 +57,6 @@ class LeaseHunter @Inject constructor(
 
   companion object {
     val LEASE_DURATION: Duration = Duration.ofMinutes(5)
+    const val RUN_PARTITION_BATCH_SIZE = 100
```

Comment:
RUN_PARTITION_QUERY_LIMIT ? This should be more than 100. At least 1000. Can we set it to 10,000 and call it a day?

Reply:
this appears to limit how many unleased partitions are fetched to try to lease them, it doesn't limit partitions per run

Reply:
If we're selecting a single random partition, why does the size of the pool matter?

Reply:
It seems we could even set `maxRows = 1`?

Reply:
maxrows=1 would mean all pods select the same one, the order is deterministic. I think 100 was fine..

Reply:
It will randomly choose over a larger set. I guess 100 is ok too. I appreciate the name change as we are doing no batching here.